handbook/engineering/roles.md

```diff
@@ -1,19 +1,20 @@
 # Roles on the Engineering team
 
-This page lists the roles on the Engineering team and some of the responsibilities for each of the roles.
+This page lists the roles on our engineering team.
 
 ## Software Engineer
 
-Software Engineers are individual contributors who build software to satisfy the needs of the business.
+Software engineers build our product and infrastructure.
 
 ### Responsibilities
 
-- Collaborate with your direct manager to identify projects that align your skills and interests with the needs of the business.
-- Write [RFCs](../communication/rfcs/index.md) to recommend solutions to product and engineering problems.
 - Commit to a reasonable amount of work that you are going to get done for each [monthly release](releases/index.md), and then reliably get it done.
+- Post weekly updates about your progress on the [monthly release tracking issue](tracking_issues.md) for your team.
```

Comment:
Should we also have some guidance here on what a weekly update looks like?

@keegancsmith had some great thoughts on the format that I can't find due to Slack history auto-delete, but paraphrased: try to provide higher-level updates instead of a list of things you did.

Good: "Last week I made progress on implementing RFC 123, but were blocked by X. X was due to Y. This week I'll continue to and hopefully implement RFC 123 completely, should be ready for review by Thurs."
Not so good: "PRs: #123, #456, #789"

Reply:
Yes we should. I just updated the tracking issue docs.

Reply:
Yep, added